tests/integration/Newsletter/ShortcodesTest.php

```diff
@@ -106,7 +106,8 @@ public function testItCanProcessCustomShortcodes() {
     $result = $shortcodesObject->process($shortcode);
     expect($result[0])->false();
     add_filter('mailpoet_newsletter_shortcode', function(
-      $shortcode, $newsletter, $subscriber, $queue, $content, $arguments) {
+      $shortcode, $newsletter, $subscriber, $queue, $content, $arguments
```

Comment:
This doesn't look good. Maybe it will be better to have the whole function declaration on one line.